src/mixer/basetrackplayer.cpp

```diff
@@ -650,6 +666,34 @@ void BaseTrackPlayerImpl::slotCloneChannel(EngineChannel* pChannel) {
     slotLoadTrack(pTrack, false);
 }
 
+void BaseTrackPlayerImpl::slotLoadTrackFromDeck(double d) {
+    int deck = static_cast<int>(d);
+    if (deck >= 1) {
```

Comment:
Does it make sense to check for the lower bound and not for the upper? I consider to remove the check.

Reply:
I've removed it.
If the group name is invalid it'll fail in `loadTrackFromGroup` anyway.